docs/releasing/before-publishing-a-release.md

```diff
@@ -12,7 +12,15 @@ Optionally, we can also let the prototype kit know that we are coordinating a re
 
 ## Raise release issues
 
-Those leading on the release should then raise new issues in the team GitHub repositories ([govuk-frontend](https://github.com/alphagov/govuk-frontend), [govuk-design-system](https://github.com/alphagov/govuk-design-system/issues/2024), [govuk-frontend-docs](https://github.com/alphagov/govuk-frontend-docs) to track the following:
+Release issues should be created in the team GitHub repositories: [govuk-frontend](https://github.com/alphagov/govuk-frontend), [govuk-design-system](https://github.com/alphagov/govuk-design-system/issues/2024) and [govuk-frontend-docs](https://github.com/alphagov/govuk-frontend-docs).
+
+All these issues should be:
+
+- added to the [Design System sprint board](https://github.com/orgs/alphagov/projects/53) backlog
+- added to that release's milestone
+- tagged with the '🚀 release' tag
+
+Those leading on the release should then raise new issues to track the following:
```

Comment:
This feels slightly out of order – as I understand it, these _are_ the release issues that are mentioned above:

'Release issues should be created in the team GitHub repositories. […] Those leading on the release should then raise new [release issues]'

Maybe these additional bullets about the projects, milestones and tags should come after the list of issues that need creating?

Reply:
Yeah, that's where they were originally. I moved them up because, in my mind, saying "each of the following issues requires these three things" makes more sense being said ahead of time rather than after-the-fact—especially if the document is being read as a set of sequential steps.

That could just be symptomatic of my failure to read ahead, though. 😛

Reply:
I think it might just be the word *then* that's throwing me off?

```suggestion
Those leading on the release should raise new issues to track the following:
```